### ISSUE TITLE
Lemon sign

### ISSUE THREAD

**Issue by azankl** [obophenotype/human-phenotype-ontology#7377]
**Preferred term label:**
Lemon sign

**Synonyms**

**Definition (free text, please give PubMed ID)**
The lemon sign refers to the shape of the fetal skull at ultrasonography (US) when the frontal bones lose their normal convex contour and appear flattened or inwardly scalloped. This gives the skull a shape similar to that of a lemon. The sign is seen on transverse sonograms of the fetal cranium obtained at the level of the ventricles.
https://pubmed.ncbi.nlm.nih.gov/12832584/

**Parent term (use hpo.jax.org/app)**
Abnormality of the calvaria HP:0002683

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
any disorder associated with neural tube defects

**Your nano-attribution (ORCID)**
Andreas Zankl 0000-0001-8612-1062
